rollup.config.ts

```diff
@@ -35,6 +35,6 @@ export default {
     resolve(),
 
     // Resolve source maps to the original source
-    sourceMaps(),
+    // sourceMaps(),
```

Comment:
lets typescript handle source map for types in build bundle, don't bother with rollup because original source isn't included anyway